src/app.ts

```diff
@@ -1,3 +1,4 @@
+import 'jquery';
```

Comment:
Curious where this uses `jquery`?

Reply:
This is an old-timey angular thing. AngularJS would use its own "lite" version of jQuery by default, but if you need regular jQuery (and there is some old-timey code in Deck that does) in your Angular modules, you have to load jQuery before Angular.